backend/lamb/owi_bridge/owi_group.py

```diff
@@ -55,6 +55,12 @@ def create_group(
             Optional[Dict]: Created group data or None if creation fails
         """
         try:
+            # Check if group with this name already exists (prevent duplicates)
```

Comment:
same here. we should assume that there aren't duplicates in the DB. This ìf` branch is not needed.